app/javascript/packages/verify-flow/steps/personal-key/download-button.tsx

```diff
@@ -15,7 +15,8 @@ interface DownloadButtonProps {
 }
 
 export interface NavigatorWithSaveBlob extends Navigator {
-  msSaveBlob: (blob: Blob, filename: string) => void;
+ //https://stackoverflow.com/questions/69485778/new-typescript-version-does-not-include-window-navigator-mssaveblob?noredirect=1&lq=1
+  msSaveBlob?: (blob: Blob, filename?: string) => boolean
```

Comment:
When running the trackEvent test for the download button I kept getting a typescript error saying the above is not a function  [this stack overflow](https://stackoverflow.com/questions/69485778/new-typescript-version-does-not-include-window-navigator-mssaveblob?noredirect=1&lq=1) offered a solution which is what I went with but maybe there is a better way to deal with this, not sure.

Reply:
Could you share the error you were seeing? The way this was implemented was meant to work around the fact that TypeScript won't expect `msSaveBlob` to exist on navigator by default, but can create a type that extends the default navigator with the function, and check for it in `hasProprietarySaveBlob`.